tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/generate_wear_study_table_test.py

```diff
@@ -115,12 +115,14 @@ def test_queries(self):
             'cleaned_values': [
                 # one yes record.
                 (1, 'Yes', self.date_one, None),
-                # two yes records. earliest recorded
-                (2, 'Yes', self.date_one, None),
-                # two yes records and one no. earliest dates recorded
-                (5, 'Yes', self.date_one, self.date_three),
+                # two yes records. latest yes recorded
+                (2, 'Yes', self.date_two, None),
+                # two no records then a yes. latest yes recorded
+                (4, 'Yes', self.date_three, None),
+                # two yes records and one no. first no record after the latest yes
+                (5, 'Yes', self.date_two, self.date_three),
                 # yes, then no, then yes. earliest dates recorded
```

Comment:
This comment should be updated too to something like:
`# yes, then no, then yes. latest dates recorded`